### PR TITLE
Re-flow Rules

### PR DESCRIPTION
**Pull request by ajyong**
PR has been updated to only include larger changes to the rules layout.  Nitpicks and a responsive nav in the header for the `default` layout to follow in future PRs.

### CODE REVIEW

**Comment by ajyong** [comment]
[ch860], I am looking for 2 approvers **minimum**.

**Comment by MarkBennett** [comment]
@ajyong is this running on Netlify? If so, can you share the link?

Otherwise, I'll hold off on reviewing until tomorrow when I can set it up on my local box.

**Review by MandyMeindersma — CHANGES_REQUESTED** [review]
can "join the fight" still be left as "home"? As a user idk where "join the fight" is taking me but I know where home is.

**Comment by ajyong** [comment]
@MarkBennett  [Netlify link](https://deploy-preview-74--codevid-19-staging.netlify.com/).  It also appears under the commit checks:
image

Netlify will also post a comment with the link from now on.  It didn't occur here because I configured that after the fact.

**Comment by MarkBennett** [comment]
It's weird, I see it as an automatic comment on some PR, and not on others @ajyong. It looks like it's just on the newer ones, so maybe it's just PRs that existed at the time you setup Netlify that are missing it?

**Comment by MandyMeindersma** [comment]
I see all the netlify links. Works on my machine :P

**Comment by ajyong** [comment]
> so maybe it's just PRs that existed at the time you setup Netlify that are missing it?

Correct.  I force-pushed some of them so they'd appear.  I also after-the-fact made Netlify create a comment to make the deploy URL more prominent.

---

> can "join the fight" still be left as "home"? As a user idk where "join the fight" is taking me but I know where home is.

I'll make this change as it's likely this will get merged before #73 (since I need to fix something in it).  I'm going to add my original reasons for removing "Home" in #73.

**Review by MandyMeindersma — APPROVED** [review]

File: _layouts/default.html
Comment (MarkBennett) [inline _layouts/default.html]: I'm not comfortable dropping `Submit Your Project` and `FAQ` from the menu simple to save space. It seems like we're compromizing function for form.

Looking at what other pages do, perhaps it's more reasonable to try a hamburger menu or similar. I don't think that simple removing them is reasonable though.

File: index.html
Comment (MarkBennett) [inline index.html]: Not part of this PR. What's the reasoning for this change?

File: policies/faq.md
Comment (MarkBennett) [inline policies/faq.md]: Again, not a big deal but I'm uncomfortable with these PRs where a lot of small things get sucked up together. It distracts from the stated purpose of this issue and makes discussion harder.

File: policies/rules.md
Comment (MarkBennett) [inline policies/rules.md:6]: I like the idea of highlighting these related items, but `FAQ` and `Submission` need to stay in the top menu.

File: policies/rules.md
Comment (MarkBennett) [inline policies/rules.md:84]: I like the call to action we had before, asking them to view details sounds like it will link to the small print and not where to go to submit.

File: policies/rules.md
Comment (MarkBennett) [inline policies/rules.md:94]: Same as above, we want to encourage people to submit more.

File: policies/rules.md
Comment (MarkBennett) [inline policies/rules.md]: I just noticed that right after `any time` the `be` should be `by`.

File: policies/rules.md
Comment (MarkBennett) [inline policies/rules.md:140]: 👍

File: policies/rules.md
Comment (MarkBennett) [inline policies/rules.md:158]: 👍

File: policies/rules.md
Comment (MarkBennett) [inline policies/rules.md:170]: 👍

File: policies/submissions.md
Comment (MarkBennett) [inline policies/submissions.md:10]: I like the new format, but let's bump this back to the top of the page. It's the first thing teams need to understand and what they'll likely want to check before submitting.

File: policies/submissions.md
Comment (MarkBennett) [inline policies/submissions.md:22]: Smart marking this as optional and keeping everything else required.

File: policies/submissions.md
Comment (MarkBennett) [inline policies/submissions.md]: I like the change to `project's placement` but let's keep the word `disadvantage` as `impact` is more obtuse.

**Review by MarkBennett — CHANGES_REQUESTED** [review]
Hey @ajyong, thanks for putting all the work in.

Reading through all the changes here, this should be two or more PRs as there's too much going on to make review easy.

I've left comments on each of the files but in general agree with most of what's in here. High priority is getting the `Submission` and `FAQ` back into the menu. I'd consider pulling the menu changes out of this PR entirely and just using this to complete the rule changes.

File: index.html
Comment (ajyong) [reply index.html]: I can revert.  Originally it was for consistency and personal preference.  I thought `<b>` tags were falling out of favour and I personally use `<strong>` over them in my own projects.  That said, both tags are valid according to our SASS.

File: policies/rules.md
Comment (ajyong) [reply policies/rules.md:84]: Except the call to action was misleading.  I read the original and even thought we had links there to our Google Forms, but all we had were just more details.  The section we take them to does not allow them to submit their project at all, hence the renaming.

File: policies/submissions.md
Comment (ajyong) [reply policies/submissions.md:10]: Agreed.  I would also opt to remove the Rules Summary section altogether and only keep a reminder to read the full rules just before the Deliverables heading.  I believe it increases the content relevancy and reduces maintenance on our part.  Thoughts?

File: policies/rules.md
Comment (ajyong) [reply policies/rules.md:6]: Is this needed anymore if we keep Submission and FAQ in the nav?

**Comment by ajyong** [comment]
Since this branch is named `reflow-rules`, I will keep it to just that.  I will split out non-relevant work into 2 other PRs:

1. Make `default` layout's nav mobile-responsive so we may link to our new top-level pages (Join the Fight and Blog)
1. Nitpicking HTML, grammatical and stylistic choices

File: _layouts/default.html
Comment (ajyong) [reply _layouts/default.html]: Reverted

File: policies/rules.md
Comment (ajyong) [reply policies/rules.md]: Small absolutely clear-as-day typos like this will be part of this PR.  Anything that's subjective I'll keep out.

File: policies/faq.md
Comment (ajyong) [inline policies/faq.md:25]: This is Prettier doing its thing as the default formatter.  It might look like a nitpick, but I'm considering it a wholesale change to fit our style guide.

File: policies/submissions.md
Comment (ajyong) [reply policies/submissions.md:10]: I know, I'll remove it from here and make it the summary in the Rules page.

**Comment by MarkBennett** [comment]
@ajyong I see you working on this. Let me know once you've resolved my previous comments and are ready for a fresh review.

**Comment by ajyong** [comment]
There are 2 comments I require input on, otherwise I have concluded the extraction of irrelevant work from this PR and it's ready for review.

File: policies/rules.md
Comment (ajyong) [reply policies/rules.md:84]: I'll go with whichever your final decision on this is @MarkBennett

**Comment by ajyong** [comment]
~~My understanding is we'll merge this PR before we also amend the rules to add tie-breaker clarifications~~

File: policies/rules.md
Comment (ajyong) [reply policies/rules.md:6]: I'll just keep it in for now.

File: policies/rules.md
Comment (ajyong) [reply policies/rules.md:84]: Still waiting on your reply here @MarkBennett

File: policies/rules.md
Comment (MarkBennett) [reply policies/rules.md:6]: I'd be fine with taking the quick links out, they're easy to add back in if needed.

File: policies/rules.md
Comment (MarkBennett) [inline policies/rules.md]: > given the opportunity award prizes

given the opportunity **to** award prizes

**Review by MarkBennett — CHANGES_REQUESTED** [review]
One tiny type that's techincally mine, not yours @ajyong! This PR looks great and the flow on the new rules page is much better.

I'd still like to draw out each individual problem a little more as it reaches that wall of text feeling after a while, but I think that's an issue for another PR.

Just add that missing "to" then I'll approve!

File: policies/rules.md
Comment (ajyong) [reply policies/rules.md]: This one slipped right by MS Word's spell/grammar checker.  It's 2020 and we have machine learning, I would've thought MS had made their checker smarter by now. 😉 

This has been fixed with the latest changeset

**Comment by ajyong** [comment]
>I'd still like to draw out each individual problem a little more as it reaches that wall of text feeling after a while, but I think that's an issue for another PR.

Yeah this was my bad.  It's more difficult to extract changes after-the-fact when they were worked on in such a coupled way.  It underscores why doing things right the first time can save everyone from a lot of headache.  I'll keep an eye on when my branches start becoming brick walls that are hard to review, but at the same time, some rare occasions will inevitably be more brick wall-esque than others.

**Pull request merged**
